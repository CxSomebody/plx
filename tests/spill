var
  a: array[256] of integer;
  x0, x1, x2, x3, x4, x5, x6, x7, x8, x9, x10, x11, x12, x13, x14, x15, x16, x17, x18, x19, x20, x21, x22, x23, x24, x25, x26, x27, x28, x29, x30, x31, x32, x33, x34, x35, x36, x37, x38, x39, x40, x41, x42, x43, x44, x45, x46, x47, x48, x49, x50, x51, x52, x53, x54, x55, x56, x57, x58, x59, x60, x61, x62, x63, x64, x65, x66, x67, x68, x69, x70, x71, x72, x73, x74, x75, x76, x77, x78, x79, x80, x81, x82, x83, x84, x85, x86, x87, x88, x89, x90, x91, x92, x93, x94, x95, x96, x97, x98, x99, x100, x101, x102, x103, x104, x105, x106, x107, x108, x109, x110, x111, x112, x113, x114, x115, x116, x117, x118, x119, x120, x121, x122, x123, x124, x125, x126, x127, x128, x129, x130, x131, x132, x133, x134, x135, x136, x137, x138, x139, x140, x141, x142, x143, x144, x145, x146, x147, x148, x149, x150, x151, x152, x153, x154, x155, x156, x157, x158, x159, x160, x161, x162, x163, x164, x165, x166, x167, x168, x169, x170, x171, x172, x173, x174, x175, x176, x177, x178, x179, x180, x181, x182, x183, x184, x185, x186, x187, x188, x189, x190, x191, x192, x193, x194, x195, x196, x197, x198, x199, x200, x201, x202, x203, x204, x205, x206, x207, x208, x209, x210, x211, x212, x213, x214, x215, x216, x217, x218, x219, x220, x221, x222, x223, x224, x225, x226, x227, x228, x229, x230, x231, x232, x233, x234, x235, x236, x237, x238, x239, x240, x241, x242, x243, x244, x245, x246, x247, x248, x249, x250, x251, x252, x253, x254, x255: integer;
  i: integer;
function f(x: integer): integer;
begin
  f := x*x;
end;
begin
  for i:=0 to 255 do a[i] := i;
  x0 := 0;
  x1 := 1;
  x2 := 8;
  x3 := 27;
  x4 := 64;
  x5 := 125;
  x6 := 216;
  x7 := 343;
  x8 := 512;
  x9 := 729;
  x10 := 1000;
  x11 := 1331;
  x12 := 1728;
  x13 := 2197;
  x14 := 2744;
  x15 := 3375;
  x16 := 4096;
  x17 := 4913;
  x18 := 5832;
  x19 := 6859;
  x20 := 8000;
  x21 := 9261;
  x22 := 10648;
  x23 := 12167;
  x24 := 13824;
  x25 := 15625;
  x26 := 17576;
  x27 := 19683;
  x28 := 21952;
  x29 := 24389;
  x30 := 27000;
  x31 := 29791;
  x32 := 32768;
  x33 := 35937;
  x34 := 39304;
  x35 := 42875;
  x36 := 46656;
  x37 := 50653;
  x38 := 54872;
  x39 := 59319;
  x40 := 64000;
  x41 := 68921;
  x42 := 74088;
  x43 := 79507;
  x44 := 85184;
  x45 := 91125;
  x46 := 97336;
  x47 := 103823;
  x48 := 110592;
  x49 := 117649;
  x50 := 125000;
  x51 := 132651;
  x52 := 140608;
  x53 := 148877;
  x54 := 157464;
  x55 := 166375;
  x56 := 175616;
  x57 := 185193;
  x58 := 195112;
  x59 := 205379;
  x60 := 216000;
  x61 := 226981;
  x62 := 238328;
  x63 := 250047;
  x64 := 262144;
  x65 := 274625;
  x66 := 287496;
  x67 := 300763;
  x68 := 314432;
  x69 := 328509;
  x70 := 343000;
  x71 := 357911;
  x72 := 373248;
  x73 := 389017;
  x74 := 405224;
  x75 := 421875;
  x76 := 438976;
  x77 := 456533;
  x78 := 474552;
  x79 := 493039;
  x80 := 512000;
  x81 := 531441;
  x82 := 551368;
  x83 := 571787;
  x84 := 592704;
  x85 := 614125;
  x86 := 636056;
  x87 := 658503;
  x88 := 681472;
  x89 := 704969;
  x90 := 729000;
  x91 := 753571;
  x92 := 778688;
  x93 := 804357;
  x94 := 830584;
  x95 := 857375;
  x96 := 884736;
  x97 := 912673;
  x98 := 941192;
  x99 := 970299;
  x100 := 1000000;
  x101 := 1030301;
  x102 := 1061208;
  x103 := 1092727;
  x104 := 1124864;
  x105 := 1157625;
  x106 := 1191016;
  x107 := 1225043;
  x108 := 1259712;
  x109 := 1295029;
  x110 := 1331000;
  x111 := 1367631;
  x112 := 1404928;
  x113 := 1442897;
  x114 := 1481544;
  x115 := 1520875;
  x116 := 1560896;
  x117 := 1601613;
  x118 := 1643032;
  x119 := 1685159;
  x120 := 1728000;
  x121 := 1771561;
  x122 := 1815848;
  x123 := 1860867;
  x124 := 1906624;
  x125 := 1953125;
  x126 := 2000376;
  x127 := 2048383;
  x128 := 2097152;
  x129 := 2146689;
  x130 := 2197000;
  x131 := 2248091;
  x132 := 2299968;
  x133 := 2352637;
  x134 := 2406104;
  x135 := 2460375;
  x136 := 2515456;
  x137 := 2571353;
  x138 := 2628072;
  x139 := 2685619;
  x140 := 2744000;
  x141 := 2803221;
  x142 := 2863288;
  x143 := 2924207;
  x144 := 2985984;
  x145 := 3048625;
  x146 := 3112136;
  x147 := 3176523;
  x148 := 3241792;
  x149 := 3307949;
  x150 := 3375000;
  x151 := 3442951;
  x152 := 3511808;
  x153 := 3581577;
  x154 := 3652264;
  x155 := 3723875;
  x156 := 3796416;
  x157 := 3869893;
  x158 := 3944312;
  x159 := 4019679;
  x160 := 4096000;
  x161 := 4173281;
  x162 := 4251528;
  x163 := 4330747;
  x164 := 4410944;
  x165 := 4492125;
  x166 := 4574296;
  x167 := 4657463;
  x168 := 4741632;
  x169 := 4826809;
  x170 := 4913000;
  x171 := 5000211;
  x172 := 5088448;
  x173 := 5177717;
  x174 := 5268024;
  x175 := 5359375;
  x176 := 5451776;
  x177 := 5545233;
  x178 := 5639752;
  x179 := 5735339;
  x180 := 5832000;
  x181 := 5929741;
  x182 := 6028568;
  x183 := 6128487;
  x184 := 6229504;
  x185 := 6331625;
  x186 := 6434856;
  x187 := 6539203;
  x188 := 6644672;
  x189 := 6751269;
  x190 := 6859000;
  x191 := 6967871;
  x192 := 7077888;
  x193 := 7189057;
  x194 := 7301384;
  x195 := 7414875;
  x196 := 7529536;
  x197 := 7645373;
  x198 := 7762392;
  x199 := 7880599;
  x200 := 8000000;
  x201 := 8120601;
  x202 := 8242408;
  x203 := 8365427;
  x204 := 8489664;
  x205 := 8615125;
  x206 := 8741816;
  x207 := 8869743;
  x208 := 8998912;
  x209 := 9129329;
  x210 := 9261000;
  x211 := 9393931;
  x212 := 9528128;
  x213 := 9663597;
  x214 := 9800344;
  x215 := 9938375;
  x216 := 10077696;
  x217 := 10218313;
  x218 := 10360232;
  x219 := 10503459;
  x220 := 10648000;
  x221 := 10793861;
  x222 := 10941048;
  x223 := 11089567;
  x224 := 11239424;
  x225 := 11390625;
  x226 := 11543176;
  x227 := 11697083;
  x228 := 11852352;
  x229 := 12008989;
  x230 := 12167000;
  x231 := 12326391;
  x232 := 12487168;
  x233 := 12649337;
  x234 := 12812904;
  x235 := 12977875;
  x236 := 13144256;
  x237 := 13312053;
  x238 := 13481272;
  x239 := 13651919;
  x240 := 13824000;
  x241 := 13997521;
  x242 := 14172488;
  x243 := 14348907;
  x244 := 14526784;
  x245 := 14706125;
  x246 := 14886936;
  x247 := 15069223;
  x248 := 15252992;
  x249 := 15438249;
  x250 := 15625000;
  x251 := 15813251;
  x252 := 16003008;
  x253 := 16194277;
  x254 := 16387064;
  x255 := 16581375;
  write(
  (((((((a[0]+a[1])+(a[2]+a[3]))+((a[4]+a[5])+(a[6]+a[7])))+(((a[8]+a[9])+(a[10]+a[11]))+((a[12]+a[13])+(a[14]+a[15]))))+((((a[16]+a[17])+(a[18]+a[19]))+((a[20]+a[21])+(a[22]+a[23])))+(((a[24]+a[25])+(a[26]+a[27]))+((a[28]+a[29])+(a[30]+a[31])))))+(((((a[32]+a[33])+(a[34]+a[35]))+((a[36]+a[37])+(a[38]+a[39])))+(((a[40]+a[41])+(a[42]+a[43]))+((a[44]+a[45])+(a[46]+a[47]))))+((((a[48]+a[49])+(a[50]+a[51]))+((a[52]+a[53])+(a[54]+a[55])))+(((a[56]+a[57])+(a[58]+a[59]))+((a[60]+a[61])+(a[62]+a[63]))))))+((((((a[64]+a[65])+(a[66]+a[67]))+((a[68]+a[69])+(a[70]+a[71])))+(((a[72]+a[73])+(a[74]+a[75]))+((a[76]+a[77])+(a[78]+a[79]))))+((((a[80]+a[81])+(a[82]+a[83]))+((a[84]+a[85])+(a[86]+a[87])))+(((a[88]+a[89])+(a[90]+a[91]))+((a[92]+a[93])+(a[94]+a[95])))))+(((((a[96]+a[97])+(a[98]+a[99]))+((a[100]+a[101])+(a[102]+a[103])))+(((a[104]+a[105])+(a[106]+a[107]))+((a[108]+a[109])+(a[110]+a[111]))))+((((a[112]+a[113])+(a[114]+a[115]))+((a[116]+a[117])+(a[118]+a[119])))+(((a[120]+a[121])+(a[122]+a[123]))+((a[124]+a[125])+(a[126]+a[127])))))))+(((((((a[128]+a[129])+(a[130]+a[131]))+((a[132]+a[133])+(a[134]+a[135])))+(((a[136]+a[137])+(a[138]+a[139]))+((a[140]+a[141])+(a[142]+a[143]))))+((((a[144]+a[145])+(a[146]+a[147]))+((a[148]+a[149])+(a[150]+a[151])))+(((a[152]+a[153])+(a[154]+a[155]))+((a[156]+a[157])+(a[158]+a[159])))))+(((((a[160]+a[161])+(a[162]+a[163]))+((a[164]+a[165])+(a[166]+a[167])))+(((a[168]+a[169])+(a[170]+a[171]))+((a[172]+a[173])+(a[174]+a[175]))))+((((a[176]+a[177])+(a[178]+a[179]))+((a[180]+a[181])+(a[182]+a[183])))+(((a[184]+a[185])+(a[186]+a[187]))+((a[188]+a[189])+(a[190]+a[191]))))))+((((((a[192]+a[193])+(a[194]+a[195]))+((a[196]+a[197])+(a[198]+a[199])))+(((a[200]+a[201])+(a[202]+a[203]))+((a[204]+a[205])+(a[206]+a[207]))))+((((a[208]+a[209])+(a[210]+a[211]))+((a[212]+a[213])+(a[214]+a[215])))+(((a[216]+a[217])+(a[218]+a[219]))+((a[220]+a[221])+(a[222]+a[223])))))+(((((a[224]+a[225])+(a[226]+a[227]))+((a[228]+a[229])+(a[230]+a[231])))+(((a[232]+a[233])+(a[234]+a[235]))+((a[236]+a[237])+(a[238]+a[239]))))+((((a[240]+a[241])+(a[242]+a[243]))+((a[244]+a[245])+(a[246]+a[247])))+(((a[248]+a[249])+(a[250]+a[251]))+((a[252]+a[253])+(a[254]+a[255])))))))
  );
  write(
  (((((((f(0)+f(1))+(f(2)+f(3)))+((f(4)+f(5))+(f(6)+f(7))))+(((f(8)+f(9))+(f(10)+f(11)))+((f(12)+f(13))+(f(14)+f(15)))))+((((f(16)+f(17))+(f(18)+f(19)))+((f(20)+f(21))+(f(22)+f(23))))+(((f(24)+f(25))+(f(26)+f(27)))+((f(28)+f(29))+(f(30)+f(31))))))+(((((f(32)+f(33))+(f(34)+f(35)))+((f(36)+f(37))+(f(38)+f(39))))+(((f(40)+f(41))+(f(42)+f(43)))+((f(44)+f(45))+(f(46)+f(47)))))+((((f(48)+f(49))+(f(50)+f(51)))+((f(52)+f(53))+(f(54)+f(55))))+(((f(56)+f(57))+(f(58)+f(59)))+((f(60)+f(61))+(f(62)+f(63)))))))+((((((f(64)+f(65))+(f(66)+f(67)))+((f(68)+f(69))+(f(70)+f(71))))+(((f(72)+f(73))+(f(74)+f(75)))+((f(76)+f(77))+(f(78)+f(79)))))+((((f(80)+f(81))+(f(82)+f(83)))+((f(84)+f(85))+(f(86)+f(87))))+(((f(88)+f(89))+(f(90)+f(91)))+((f(92)+f(93))+(f(94)+f(95))))))+(((((f(96)+f(97))+(f(98)+f(99)))+((f(100)+f(101))+(f(102)+f(103))))+(((f(104)+f(105))+(f(106)+f(107)))+((f(108)+f(109))+(f(110)+f(111)))))+((((f(112)+f(113))+(f(114)+f(115)))+((f(116)+f(117))+(f(118)+f(119))))+(((f(120)+f(121))+(f(122)+f(123)))+((f(124)+f(125))+(f(126)+f(127))))))))+(((((((f(128)+f(129))+(f(130)+f(131)))+((f(132)+f(133))+(f(134)+f(135))))+(((f(136)+f(137))+(f(138)+f(139)))+((f(140)+f(141))+(f(142)+f(143)))))+((((f(144)+f(145))+(f(146)+f(147)))+((f(148)+f(149))+(f(150)+f(151))))+(((f(152)+f(153))+(f(154)+f(155)))+((f(156)+f(157))+(f(158)+f(159))))))+(((((f(160)+f(161))+(f(162)+f(163)))+((f(164)+f(165))+(f(166)+f(167))))+(((f(168)+f(169))+(f(170)+f(171)))+((f(172)+f(173))+(f(174)+f(175)))))+((((f(176)+f(177))+(f(178)+f(179)))+((f(180)+f(181))+(f(182)+f(183))))+(((f(184)+f(185))+(f(186)+f(187)))+((f(188)+f(189))+(f(190)+f(191)))))))+((((((f(192)+f(193))+(f(194)+f(195)))+((f(196)+f(197))+(f(198)+f(199))))+(((f(200)+f(201))+(f(202)+f(203)))+((f(204)+f(205))+(f(206)+f(207)))))+((((f(208)+f(209))+(f(210)+f(211)))+((f(212)+f(213))+(f(214)+f(215))))+(((f(216)+f(217))+(f(218)+f(219)))+((f(220)+f(221))+(f(222)+f(223))))))+(((((f(224)+f(225))+(f(226)+f(227)))+((f(228)+f(229))+(f(230)+f(231))))+(((f(232)+f(233))+(f(234)+f(235)))+((f(236)+f(237))+(f(238)+f(239)))))+((((f(240)+f(241))+(f(242)+f(243)))+((f(244)+f(245))+(f(246)+f(247))))+(((f(248)+f(249))+(f(250)+f(251)))+((f(252)+f(253))+(f(254)+f(255))))))))
  );
  write(
  (((((((x0+x1)+(x2+x3))+((x4+x5)+(x6+x7)))+(((x8+x9)+(x10+x11))+((x12+x13)+(x14+x15))))+((((x16+x17)+(x18+x19))+((x20+x21)+(x22+x23)))+(((x24+x25)+(x26+x27))+((x28+x29)+(x30+x31)))))+(((((x32+x33)+(x34+x35))+((x36+x37)+(x38+x39)))+(((x40+x41)+(x42+x43))+((x44+x45)+(x46+x47))))+((((x48+x49)+(x50+x51))+((x52+x53)+(x54+x55)))+(((x56+x57)+(x58+x59))+((x60+x61)+(x62+x63))))))+((((((x64+x65)+(x66+x67))+((x68+x69)+(x70+x71)))+(((x72+x73)+(x74+x75))+((x76+x77)+(x78+x79))))+((((x80+x81)+(x82+x83))+((x84+x85)+(x86+x87)))+(((x88+x89)+(x90+x91))+((x92+x93)+(x94+x95)))))+(((((x96+x97)+(x98+x99))+((x100+x101)+(x102+x103)))+(((x104+x105)+(x106+x107))+((x108+x109)+(x110+x111))))+((((x112+x113)+(x114+x115))+((x116+x117)+(x118+x119)))+(((x120+x121)+(x122+x123))+((x124+x125)+(x126+x127)))))))+(((((((x128+x129)+(x130+x131))+((x132+x133)+(x134+x135)))+(((x136+x137)+(x138+x139))+((x140+x141)+(x142+x143))))+((((x144+x145)+(x146+x147))+((x148+x149)+(x150+x151)))+(((x152+x153)+(x154+x155))+((x156+x157)+(x158+x159)))))+(((((x160+x161)+(x162+x163))+((x164+x165)+(x166+x167)))+(((x168+x169)+(x170+x171))+((x172+x173)+(x174+x175))))+((((x176+x177)+(x178+x179))+((x180+x181)+(x182+x183)))+(((x184+x185)+(x186+x187))+((x188+x189)+(x190+x191))))))+((((((x192+x193)+(x194+x195))+((x196+x197)+(x198+x199)))+(((x200+x201)+(x202+x203))+((x204+x205)+(x206+x207))))+((((x208+x209)+(x210+x211))+((x212+x213)+(x214+x215)))+(((x216+x217)+(x218+x219))+((x220+x221)+(x222+x223)))))+(((((x224+x225)+(x226+x227))+((x228+x229)+(x230+x231)))+(((x232+x233)+(x234+x235))+((x236+x237)+(x238+x239))))+((((x240+x241)+(x242+x243))+((x244+x245)+(x246+x247)))+(((x248+x249)+(x250+x251))+((x252+x253)+(x254+x255)))))))
  );
end.
