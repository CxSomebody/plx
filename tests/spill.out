32640
5559680
1065369600
